docs/api/platform/FileHandle.md

```diff
@@ -25,40 +25,39 @@ API                            | C/C++                   | POSIX
 Sigio                          | No                      | No                     | Yes
-Device-specific extensions     | No                      | No                     | Possible via derived types
-Newline conversion             | Yes (enabled via JSON)  | No                     | No
+Device-specific extensions     | No                      | No                     | Possible using derived types
+Newline conversion             | Yes (enabled with JSON) | No                     | No
 Error indications              | EOF, ferror, set errno  | Return -1, set errno   | Return negative error code
 Portability                    | High                    | Medium                 | Low
 
-The APIs can be mixed with care, for example setting up a callback initially with `FileHandle::sigio`, but performing all subsequent operations using POSIX.
+You can mix the APIs if you're careful, for example setting up a callback initially with `FileHandle::sigio` but performing all subsequent operations using POSIX.
 
-* Note: `errno` is not currently thread-local on all toolchains. This may cause problems with error handling if multiple threads are using POSIX or C file APIs simultaneously.
+<span class="notes">**Note:** `errno` is not thread-local on all toolchains. This may cause problems with error handling if multiple threads are using POSIX or C file APIs simultaneously.</span>
  
 ### Mapping between APIs
 
 Calls are provided to attach already-opened lower levels to the higher levels:
 
-* `int mbed_bind_to_fd(FileHandle *)` bind a FileHandle to a POSIX file descriptor
-* `FILE *fdopen(int fd, const char *mode)` bind a POSIX file descriptor to a stdio FILE
-* `FILE *fdopen(FileHandle *fh, const char *mode)` bind a FileHandle to a stdio FILE
+- `int mbed_bind_to_fd(FileHandle *)` bind a FileHandle to a POSIX file descriptor.
+- `FILE *fdopen(int fd, const char *mode)` bind a POSIX file descriptor to a stdio FILE.
+- `FILE *fdopen(FileHandle *fh, const char *mode)` bind a FileHandle to a stdio FILE.
 
-There is currently no call to map from POSIX file descriptor to `FileHandle`; this may be added in future.
+The standard POSIX function `int fileno(FILE *stream)` may be available to map from `FILE` to file descriptor, depending on the toolchain and C library in use - it is not usable in fully portable Mbed OS code.
 
-The standard POSIX function `int fileno(FILE *stream)` may be available to map from `FILE` to file descriptor, depending on the toolchain and C library in use - it is not useable in fully-portable Mbed OS code.
-
-Given those limitations on mapping, if code needs to access the lower levels, a lower-level open call should be used, so the  lower-level handle is known, then that is bound to the higher level(s).
+Given those limitations on mapping, if code needs to access the lower levels, use a lower-level open call, so the lower-level handle is known. Then, that is bound to the higher level.
```

Comment:
This paragraph is dangling a bit with the "currently no call to map from POSIX fd to FileHandle" paragraph gone.

Change the intro to "As it is not possible to map from higher levels to lower levels,".

Final sentence should be imperative to match previous. "Then bind that to the higher level."